cmd/jujud/agent/machine/manifolds.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/juju/proxy"
 	"github.com/juju/pubsub"
 	"github.com/juju/utils/clock"
+	utilsfeatureflag "github.com/juju/utils/featureflag"
```

Comment:
Why the alias when it's not needed?

Reply:
There is a "worker/featureflag" in there which will conflict.